<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>form_bug</class>
 <widget class="QWidget" name="form_bug">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>720</width>
    <height>510</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="main_vlayout">
   <property name="spacing">
    <number>15</number>
   </property>
   <property name="leftMargin">
    <number>30</number>
   </property>
   <property name="topMargin">
    <number>10</number>
   </property>
   <property name="rightMargin">
    <number>30</number>
   </property>
   <property name="bottomMargin">
    <number>25</number>
   </property>
   <item>
    <widget class="QLabel" name="lbl_bug">
     <property name="font">
      <font>
       <pointsize>24</pointsize>
      </font>
     </property>
     <property name="text">
      <string>[New/Edit bug]</string>
     </property>
     <property name="wordWrap">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="hlayout_status">
     <property name="spacing">
      <number>15</number>
     </property>
     <property name="rightMargin">
      <number>10</number>
     </property>
     <item>
      <widget class="QLabel" name="lbl_status">
       <property name="text">
        <string>Status:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="le_status">
       <property name="cursor">
        <cursorShape>ArrowCursor</cursorShape>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="placeholderText">
        <string>[Status]</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="lbl_system">
       <property name="text">
        <string>System:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="system_le">
       <property name="text">
        <string/>
       </property>
       <property name="placeholderText">
        <string>[System]</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="lbl_version">
       <property name="text">
        <string>System Version:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="le_version">
       <property name="text">
        <string/>
       </property>
       <property name="placeholderText">
        <string>[System Version]</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="lbl_id">
       <property name="text">
        <string>Service now ID:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="le_id">
       <property name="text">
        <string/>
       </property>
       <property name="placeholderText">
        <string>[Service now ID]</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="hlayout_date" stretch="0,0,0,0">
     <property name="spacing">
      <number>15</number>
     </property>
     <property name="rightMargin">
      <number>150</number>
     </property>
     <item>
      <widget class="QLabel" name="lbl_creation">
       <property name="text">
        <string>Creation time:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="le_creation">
       <property name="text">
        <string/>
       </property>
       <property name="placeholderText">
        <string>[Creation time]</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="lbl_update">
       <property name="text">
        <string>Last update:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="le_update">
       <property name="text">
        <string/>
       </property>
       <property name="placeholderText">
        <string>[Last update]</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="hlayout_campaign">
     <property name="spacing">
      <number>15</number>
     </property>
     <property name="rightMargin">
      <number>275</number>
     </property>
     <item>
      <widget class="QLabel" name="lbl_campaign">
       <property name="text">
        <string>Campaign:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QComboBox" name="cb_campaign"/>
     </item>
     <item>
      <widget class="QLabel" name="lbl_requirements">
       <property name="text">
        <string>Requirements affected:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QComboBox" name="cb_requirements"/>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="hlayout_urgency">
     <property name="spacing">
      <number>15</number>
     </property>
     <property name="rightMargin">
      <number>275</number>
     </property>
     <item>
      <widget class="QLabel" name="lbl_urgency">
       <property name="text">
        <string>Urgency:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QComboBox" name="cb_urgency"/>
     </item>
     <item>
      <widget class="QLabel" name="lbl_impact">
       <property name="text">
        <string>Impact:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QComboBox" name="cb_impact"/>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="hlayout_description">
     <property name="rightMargin">
      <number>10</number>
     </property>
     <item>
      <widget class="QLabel" name="lbl_short">
       <property name="text">
        <string>Short description: </string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="le_short">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="alignment">
        <set>Qt::AlignmentFlag::AlignLeading|Qt::AlignmentFlag::AlignLeft|Qt::AlignmentFlag::AlignTop</set>
       </property>
       <property name="placeholderText">
        <string>[Short decription]</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="hlayout_definition">
     <property name="spacing">
      <number>10</number>
     </property>
     <property name="topMargin">
      <number>5</number>
     </property>
     <property name="rightMargin">
      <number>10</number>
     </property>
     <property name="bottomMargin">
      <number>50</number>
     </property>
     <item>
      <widget class="QLabel" name="lbl_definition">
       <property name="text">
        <string>Definition:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="le_definition">
       <property name="text">
        <string/>
       </property>
       <property name="alignment">
        <set>Qt::AlignmentFlag::AlignLeading|Qt::AlignmentFlag::AlignLeft|Qt::AlignmentFlag::AlignTop</set>
       </property>
       <property name="placeholderText">
        <string>[Definition]</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="hlayout_btn_bug">
     <property name="spacing">
      <number>5</number>
     </property>
     <property name="leftMargin">
      <number>400</number>
     </property>
     <item>
      <widget class="QPushButton" name="btn_add_bug">
       <property name="text">
        <string>Create bug</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="btn_cancel">
       <property name="text">
        <string>Cancel</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
